common/tests/test_middleware.py

```
@@ -0,0 +1,129 @@
+        client_id = "my-client-request-id-123"
+        request = self.factory.get("/", HTTP_X_REQUEST_ID=client_id)
+        mw, get_response, response = _make_middleware()
+
+        mw(request)
+
+        self.assertEqual(request.request_id, client_id)
+
+    def test_missing_header_generates_uuid(self):
+        """No X-Request-ID header → middleware generates one."""
+        request = self.factory.get("/")
+        mw, get_response, response = _make_middleware()
```

Ruff (0.15.10)

[warning] Unpacked variable `get_response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)